### ISSUE TITLE
Evaluate features normalization

### ISSUE THREAD

**Issue by gcroci2** [DeepRank/3D-Vac#91]
Scaling input and output variables is a critical step in using neural network models, and in practice it is nearly always advantageous. Unscaled input variables can result in a slow or unstable learning process, whereas unscaled target variables on regression problems can result in exploding gradients causing the learning process to fail (see [this blog post](https://machinelearningmastery.com/how-to-improve-neural-network-stability-and-modeling-performance-with-data-scaling/) as reference).

We should evaluate performances with and without features scaling. In particular, referring to [Sklearn methods](https://towardsdatascience.com/scale-standardize-or-normalize-with-scikit-learn-6ccc7d176a02), we may try:
- MinMaxScaler
- RobustScaler
- Normalizer
- StandardScaler

We should also add this info to the experiments_log.xlsx file
